From: vojtech@suse.cz
Subject: [PATCH 3/3] Fix 'event field not found' message filling logs

 ===================================================================
 ChangeSet@1.1977.1.3, 2005-01-29 13:09:24+01:00, vojtech@silver.ucw.cz
   input: Ignore non-LED events in hid-input hidinput_event(). This gets rid
          of the "event field not found" message caused by EV_MSC type events.

   Signed-off-by: Vojtech Pavlik <vojtech@suse.cz>


  hid-input.c |    3 +++
  1 files changed, 3 insertions(+)
 ===================================================================

diff -purN linux-2.6.11-rc2-bk9/drivers/usb/input/hid-input.c linux-2.6.11-rc2-bk10/drivers/usb/input/hid-input.c
--- linux-2.6.11-rc2-bk9/drivers/usb/input/hid-input.c	2005-02-01 16:06:19.758769398 +0100
+++ linux-2.6.11-rc2-bk10/drivers/usb/input/hid-input.c	2005-02-01 16:28:23.089001327 +0100
@@ -492,6 +492,9 @@ static int hidinput_input_event(struct i
 	if (type == EV_FF)
 		return hid_ff_event(hid, dev, type, code, value);
 
+	if (type != EV_LED)
+		return -1;
+
 	if ((offset = hid_find_field(hid, type, code, &field)) == -1) {
 		warn("event field not found");
 		return -1;
